### ISSUE TITLE
Arbitrum local development network name inconsistency

### ISSUE THREAD

**Issue by devey** [graphprotocol/graph-cli#975]
I was trying to setup the [Subgraph Debug Forking][1] for **Arbitrum** and ended up having the same error:
```bash
Failed to deploy to Graph node http://127.0.0.1:8020/: Ethereum network not supported by registrar: mainnet
```

This issue got fixed by using:
```yaml
 network: arbitrum
```
instead of:
```yaml
 network: arbitrum-one
```
in `subgraph.yaml`.

Not sure why as the second one is the correct value (might have to do with the fact this forking functionality is developed by a contractor company).

  [1]: https://thegraph.com/docs/en/cookbook/subgraph-debug-forking/

**Comment by azf20** [comment]
hey @devey - interesting, were you running a local graph-node?

**Comment by devey** [comment]
@azf20 correct, just followed these steps on [Running a local Graph Node](https://github.com/graphprotocol/graph-node#running-a-local-graph-node).

**Comment by azf20** [comment]
I think this relates to the name of the network configured on the local Graph Node (which must match the name in the subgraph.yaml). Closing for now, please re-open if this issue is ongoing
